src/java/org/apache/cassandra/repair/consistent/LocalSessions.java

```diff
@@ -406,7 +406,9 @@ private void syncTable()
     {
         TableId tid = Schema.instance.getTableMetadata(keyspace, table).id;
         ColumnFamilyStore cfm = Schema.instance.getColumnFamilyStoreInstance(tid);
-        cfm.forceBlockingFlush();
```

Comment:
I'm not sure this check is ever required, in fact `keyspace` and `table` are hardcoded and always exist:
```
private final String keyspace = SchemaConstants.SYSTEM_KEYSPACE_NAME;
private final String table = SystemKeyspace.REPAIRS;
```